src/QsCompiler/CompilationManager/CompilationUnit.cs

```diff
@@ -483,25 +483,29 @@ private QsCallable GetImportedCallable(CallableDeclarationHeader header)
         {
             // TODO: this needs to be adapted if we want to support external specializations
             if (header == null) throw new ArgumentNullException(nameof(header));
-            var importedSpecs = this.GlobalSymbols.ImportedSpecializations(header.QualifiedName);
             if (Namespace.IsDeclarationAccessible(false, header.Modifiers.Access))
             {
-                var definedSpecs = this.GlobalSymbols.DefinedSpecializations(header.QualifiedName);
+                var definedSpecs = GlobalSymbols.DefinedSpecializations(header.QualifiedName);
```

Comment:
nit/just something to think about: I kind of like the explicit "this" since it makes it transparent what are instance methods and which ones are static.

Reply:
I added it back.